ecl/hql/hqlutil.cpp

```diff
@@ -8079,6 +8079,8 @@ class Vs6CppNameMangler
             mangled.append("PVIGlobalCodeContext@@");
         else if (body->hasAttribute(userMatchFunctionAtom))
             mangled.append("PVIMatchWalker@@");
+        if (functionBodyIsActivity(body))
```

Comment:
Not familiar with the mangling schemes, but is it correct that this one is an if while the others are "else if" ?

Reply:
Because the others seem to be exclusive options - e.g., see generateFunctionPrototype().  I was missing an entry for windows name mangling.